inspire/dojson/common/base.py

```diff
@@ -39,6 +39,12 @@ def control_number(self, key, value):
     return value[0]
 
 
+@hep2marc.over('001', 'control_number')
+def control_number2marc(self, key, value):
+    """Record Identifier."""
+    return [value]
```

Comment:
Not really, what confused me was that:

```python
In [1]: from dojson.contrib.marc21.utils import create_record, split_blob

In [2]: for data in split_blob(open("/Users/jmartinm/Desktop/xm.xml").read()):
    record = create_record(data)
   ...:

In [3]: record
Out[3]:
{'001': ['1381023'],
 '005': ['20150714113421.0'],
 u'020__': [{'a': '978-3-319-18335-0', 'b': 'online'},
  {'a': '978-3-319-18334-3', 'b': 'print'}],
 u'035__': {'9': 'INSPIRETeX', 'a': 'Proceedings:2015hya'},
 u'100__': {'a': u'P\xfctzfeld, Dirk', 'e': 'ed.'},
 u'245__': {'a': 'Proceedings, 524th WE-Heraeus-Seminar: Equations of Motion in Relativistic Gravity (EOM 2013)',
  'b': 'Bad Honnef, Germany, February 17-23, 2013'},
```

Maybe https://github.com/inveniosoftware/dojson/blob/52c052717a175098331d1224e42b7747777eb764/dojson/contrib/marc21/utils.py#L27 should be amended to not produce a list for the controlfields or is intended?